server/fridgeServer.go

```diff
@@ -0,0 +1,175 @@
+package server
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/ASeegull/SmartFridge/server/database"
+	"github.com/gorilla/mux"
+	"github.com/gorilla/websocket"
+	"gopkg.in/yaml.v2"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	defaultPort = "9000"
+	defaultHost = "localhost"
+)
+
+type config struct {
+	Port string `yaml:"port"`
+	Host string `yaml:"host"`
+}
+
+var serverConfig config
+
+func (config *config) getConf() *config {
+
+	yamlFile, err := ioutil.ReadFile("../SmartFridge/fridgeServerConfig.yaml")
```

Comment:
The path is relative to directory, where the process is initialized (/server in this case). Your path to config file should look like "../../fridgeServerConfig.yaml"